src/arch/sparcv9/CodeGen.zig

```diff
@@ -892,6 +892,12 @@ fn airCall(self: *Self, inst: Air.Inst.Index, modifier: std.builtin.CallOptions.
                         },
                     },
                 });
+
+                // TODO Find a way to fill this slot
+                _ = try self.addInst(.{
+                    .tag = .nop,
+                    .data = .{ .nop = {} },
+                });
```

Comment:
Is this related to https://arcb.csc.ncsu.edu/~mueller/codeopt/codeopt00/notes/delaybra.html? If so it's probably a good idea to have it linked here so that it does not get forgotten.

Reply:
Yes, the nops are there to fill the delay slot with something safe.
For the time being it's needed to be done everytime we emit a branch, do I need to add the note in all of them, or?

Reply:
I'm not quite sure honestly. Perhaps it's a good idea to add that link to the first occurrence and leave the rest without the link.

Reply:
Oh I forgot to say, so for this I put a comment in the `return` instruction in the epilogue but for the rest I'm keeping it as-is. Is that okay with you?